backend/discord_bot/steward_commands.py

```
@@ -31,40 +29,9 @@
-
-    role_ids = {
-        str(role_id)
-        for role_id in ((interaction.get('member') or {}).get('roles') or [])
-    }
-    if AUTHORIZED_ROLE_IDS.isdisjoint(role_ids):
-        return None, channel_message(
-            'Only Community Core and Onboarding Core members can manage Community Points.',
-            ephemeral=True,
-        )
 
+def _linked_invoker(interaction):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_linked_invoker`

(ANN202)